MAINTENANCE.md

```diff
@@ -61,6 +61,15 @@ repository to avoid duplicating work across multiple repositories.
 If you find any problems with the test setup and deployment, please create issues and
 submit pull requests to that repository.
 
+## Continuous Documentation
+
+We use the [Zeit Now for Github integration](https://zeit.co/github) to preview changes
+made to our documentation website every time we make a commit in a pull request.
+The integration service has a configuration file `now.json`, with a list of options to
+change the default behaviour at https://zeit.co/docs/configuration.
+The actual script `package.json` is used by Zeit Now to install the necessary packages,
+build the documentation, copy the files to a 'public' folder and deploy that to the web,
+see https://zeit.co/docs/v2/build-step/?query=package.json#defining-a-build-script.
```

Comment:
> @weiji14 i thought that might be the case. Don’t worry too much about it. What you can do is include a section in `MAINTENANCE.md` which briefly explains the setup, files involved, and links to services so we can easily find them.

Ok, the comments in JSON definitely doesn't work, so I've added this 'Continuous Documentation' section to 'MAINTENANCE.md', just below 'Continuous Integration'. Hopefully that's enough for now.